sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/io/sermant/router/dubbo/handler/LaneContextFilterHandler.java

```diff
@@ -77,6 +80,10 @@ public Map<String, List<String>> getRequestTag(Object invoker, Object invocation
         if (LOGGER.isLoggable(Level.FINE)) {
             LOGGER.fine("Lane is " + requestTag);
         }
+        MetricsManager.addOrUpdateMetricValue(RouterConstant.DYEING_COUNT, RouterConstant.LANE_TAG_NAME,
```

Comment:
donot use dyeing since we have been using lane

Reply:
done